bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,325 @@
+                        await channel.send(f"Your words were: {wrds}!")
+
+                    else:  # there is only one winner
+                        # There cant be 0 winners since we are checking that above
+
+                        winner = winners[0][1]
+                        await channel.send(f"{winner['author'].mention}!")
+                        await channel.send(f"Congratulations {winner['author'].mention}! You have a score of "
+                                           f"{winner['score']}!")
+                        await channel.send(f"Your word was: **{winner['word']}**.")  # display winner, word and score
+
+                    # Send random party emojis
+                    party = [random.choice([':partying_face:', ':tada:']) for _ in range(random.randint(1, 10))]
+                    await channel.send(" ".join(party))
+
+                else:
+                    # no participants in current iteration
+                    await channel.send('Hmm... Looks like no one participated! :cry:')
+
+            self.messages = {}  # reset the messages
+
+        # send the next name
+        self.word = f"{random.choice(self.first_names)} {random.choice(self.last_names)}"
+
+        await channel.send("Let's move on to the next name!\nAnd the next name is...\n"
+                           f"**{self.word}**!\nTry to spookify that... :smirk:")
+
+        self.poll = False  # accepting responses
+
+    async def get_responses_list(self, final: bool = False) -> discord.Embed:
+        """Returns an embed containing the responses of the people."""
+        channel = await self.get_channel()
+
+        embed = discord.Embed(
+            color=Colours.red,
+            title="",
+            description=""
+        )
+
+        if self.messages:
+            if final:
+                embed.title = "Spook Name Rate is about to end!"
+                embed.description = ("This Spook Name Rate round is about to end in 2 hours! You can review "
+                                     "the entries below! Have you rated other's words?")
+            else:
+                embed.title = "All the spookified names!"
+                embed.description = "See a list of all the entries entered by everyone!"
+        else:
+            embed.title = "No one has added an entry yet..."
+
+        for message in self.messages:
+            data = self.messages.get(message)
+
+            embed.add_field(
+                name=data['author'],
+                value=f"[{data['word']}](https://discord.com/channels/{Client.guild}/{channel.id}/{message})",
+            )
+
+        return embed
+
+    async def get_channel(self) -> TextChannel:
+        """Gets the seasonalbot-channel after waiting until ready."""
+        await self.bot.wait_until_ready()
+        return self.bot.get_channel(self.channel_id)
+
+    @staticmethod
+    def load_json(file: Path) -> dict:
```

Comment:
```suggestion
    def load_json(file: Path) -> Dict:
```
Use typing.Dict, preferably with subtypes if they are easy enough to type, like Dict[str, str] if it's string keys, mapping to string values